src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,54 @@
+.stopwatch {
+  box-sizing: border-box;
+  position: relative;
+  width: $stopwatchSize;
+  height: $stopwatchSize;
+  justify-content: center;
+  align-items: center;
+  
+  &--face {
```

Comment:
The modifier is not really needed here since we are not using the unmodified version of the `stopwatch` class. We could include those rules inside `stopwatch` class itself.

Reply:
Yes, that's right :) I didn't notice at the end that it was unnecessary. Thank you :)